test/ttmlir/Conversion/TosaToTTIR/matmul_op.mlir

```diff
@@ -0,0 +1,11 @@
+// RUN: ttmlir-opt --convert-tosa-to-ttir %s | FileCheck %s
+module attributes {} {
+  func.func @test_matmul(%arg0: tensor<13x21x16xf32>, %arg1: tensor<13x16x31xf32>) -> tensor<13x21x31xf32> {
+    // CHECK: func.func {{.+}}%arg{{[0-9]+}}: tensor<[[B:[0-9]+]]x[[I:[0-9]+]]x[[J:[0-9]+]]xf32>, %arg{{[0-9]+}}: tensor<[[B:[0-9]+]]x[[J:[0-9]+]]x[[K:[0-9]+]]xf32>
```

Comment:
Nitpick: `func.func {{.+}}%arg{{` add space before `%`.

Really precise and verbose `CHECK`s. Good job. I [repeat](https://github.com/tenstorrent/tt-mlir/pull/1480#discussion_r1869082233) that we should fix all tests to be like this, since many old ones do not use this level of details for `CHECK`s.

Reply:
That causes the test to fail(verified locally). The space becomes a part of the pattern and then it can no longer match "(%arg0".